CODEOWNERS

```diff
@@ -112,7 +112,7 @@
 /bundles/org.openhab.binding.onewiregpio/ @aogorek
 /bundles/org.openhab.binding.onewire/ @J-N-K
 /bundles/org.openhab.binding.onkyo/ @pail23 @paulianttila
-/bundles/org.openhab.binding.opensprinkler/ @CrackerStealth
+/bundles/org.openhab.binding.opensprinkler/ @CrackerStealth @FlorianSW
```

Comment:
@kaikreuzer Can you please add @FlorianSW to the contributors team? Thank you.

Reply:
Done.